### PR TITLE
indicate in the UI which content is community edition

### PR DESCRIPTION
**Pull request by MrGrinst**
image
image
image

### CODE REVIEW

File: src/routes/view-content/[guideId]/[bibleSection]/feedback/ResourceFeedbackModal.svelte [hunk 1]
```diff
@@ -34,8 +34,8 @@
     }
 
     function close() {
-        resetForm();
         closeResourceFeedbackModal();
+        setTimeout(resetForm, 250); // wait for animation to end before resetting the form so it doesn't flash
```
Comment (MrGrinst) [inline src/routes/view-content/[guideId]/[bibleSection]/feedback/ResourceFeedbackModal.svelte:38]: Added this as a fix for a flash I was noticing.

File: src/routes/view-content/[guideId]/[bibleSection]/library-resource-menu/FullscreenTextResource.svelte [hunk 2]
```diff
@@ -92,12 +95,24 @@
...
                 </div>
             </div>
-            <div class="prose mx-auto overflow-y-scroll px-4 pb-4 md:px-0 {!!resource?.audioUrl && 'mb-14'}">
-                {#if resource?.html}
-                    {@html resource.html}
-                {:else}
-                    {$translate('page.passage.resourcePane.error.value')}
+            <div class="self-center overflow-y-scroll">
+                {#if resource?.communityEdition}
+                    <div class="float-right p-4 pe-5 text-warning">
```
Comment (jwinston-bn) [inline src/routes/view-content/[guideId]/[bibleSection]/library-resource-menu/FullscreenTextResource.svelte:100]: Does `float-right` behave as expected for RTL?

Comment (MrGrinst) [reply src/routes/view-content/[guideId]/[bibleSection]/library-resource-menu/FullscreenTextResource.svelte:100]: Great question, I will test that

Comment (MrGrinst) [reply src/routes/view-content/[guideId]/[bibleSection]/library-resource-menu/FullscreenTextResource.svelte:100]: It needs to be `float-end`, thanks for catching!

**Review by jwinston-bn — APPROVED** [review]

**Pull request merged**